### ISSUE TITLE
Leverage browser caching

### ISSUE THREAD

**Issue by coliff** [hnkansai/hnkansai.github.io#14]
> Setting an expiry date or a maximum age in the HTTP headers for static resources instructs the browser to load previously downloaded resources from local disk rather than over the network.

Currently, most images have a 10 minute cache life.

https://developers.google.com/speed/pagespeed/insights/?url=hnkansai.github.io&tab=mobile

**Comment by benmurden** [comment]
Can this even be done with GitHub Pages? One would need access to set HTTP headers, which as far as I know, can't be done here.

**Comment by coliff** [comment]
Good point. Would need to add Cloudflare for this then.

**Comment by benmurden** [comment]
Can that be done on the free tier? Do you have experience setting that up?

**Comment by coliff** [comment]
Yes, I've set it up a number of times. It's totally free and has many advantages - including free SSL, so the URL can be: https://hnkansai.org/. I'll post the required steps here shortly....

**Comment by coliff** [comment]
Quick instructions here. Really easy, setup free Cloudflare account, change Nameservers for the domain to ones provided by Cloudflare. 
https://hackernoon.com/set-up-ssl-on-github-pages-with-custom-domains-for-free-a576bdf51bc
